src/public/log/schema.rs

```diff
@@ -1075,17 +1078,20 @@ mod test {
         policy_ids.insert(PolicyId::from_str("policy1").unwrap());
         policy_ids.insert(PolicyId::from_str("policy2").unwrap());
 
-        let errors = (0..num_of_error)
-            .map(|i| AuthorizationError::PolicyEvaluationError {
-                id: PolicyID::from_string(format!("policy{i}")),
-                error: EvaluationError::from(RestrictedExprError::InvalidRestrictedExpression {
-                    feature: Default::default(),
-                    expr: Value::from(true).into(),
-                }),
-            })
-            .collect();
-
-        Response::new(decision, policy_ids, errors)
+        // Old code - no longer works
+        // let errors = (0..num_of_error)
+        //     .map(|i| AuthorizationError::PolicyEvaluationError {
+        //         id: PolicyID::from_string(format!("policy{i}")),
+        //         error: EvaluationError::from(RestrictedExpr::InvalidRestrictedExpression {
+        //             feature: Default::default(),
+        //             expr: Value::from(true).into(),
+        //         }),
+        //     })
+        //     .collect();
+        println!("Number of errors needed: {}", { num_of_error });
+        // Uses a empty vector now instead of giving num_of_error errors. Tests have been changed to reflect this
+        // Leads to problems in test coverage
```

Comment:
What's the problem in test coverage? Also, can you remove these comments?

Reply:
There's tests testing for different number of errors, which are completely useless now that we can't create the exact error. It was suggested to me that I just use a blank vector for everything, which might cause issues?

Reply:
Are there specific lines of code that are now uncovered by removing those test cases?

Reply:
Nope, these are the lines of code it's supposed to cover, so removing these makes the test cases obsolete. I'm hesitant to do it though because I don't quite understand why the test cases are done the way they are

Reply:
removed